### PR TITLE
Train test

### PR DESCRIPTION
**Pull request by mkpurcell**

### CODE REVIEW

File: examples/ffl/register.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+LOGGER = logging.getLogger(__package__)
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Messaging Client')
+    parser.add_argument('--credentials', required=True)
+    parser.add_argument('--user', help='A new user account')
+    parser.add_argument('--password', help='Password for new user account')
+    parser.add_argument('--org', required=False, default='Musketeer', help='User organisation')
```
Comment (ambrishrawat) [inline examples/ffl/register.py]: Should we change the default to 'IBM'?

File: tests/train.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+            #Should fail because of ACL
+            with self.assertRaises(fflabc.ServerException):
+                result = worker.delete_model(self.task)
+
+    def aggr_wait(self, aggr, worker_id, wanted, timeout=0.1):
+        #Aggregator enters listening mode and waits for notifications
+        with aggr:
+            result = aggr.receive(timeout)
+            LOGGER.debug(f"Aggregator received: {result.content}")
+            #LOGGER.info(f"Aggregator received: {result.notification}")
```
Comment (ambrishrawat) [inline tests/train.py]: Perhaps clean up the commented LOGGER commands?

File: examples/ffl/register.py (hunk 1 above)
Comment (mkpurcell) [reply examples/ffl/register.py]: Yep

File: tests/train.py (hunk 2 above)
Comment (mkpurcell) [reply tests/train.py]: Cool.

**Review by ambrishrawat — APPROVED** [review]

**Review by ambrishrawat — APPROVED** [review]

**Pull request merged**